src/ggrc/converters/column_handlers.py

```diff
@@ -4,23 +4,23 @@
 """
 This module provides all column handlers for objects in the ggrc module.
 
-If you want to add column handler you should decide is it handler default
+If you want to add column handler you should decide is it default handler
 or custom for current model.
 If this handler is default than you will add it into _COLUMN_HANDLERS dict in
 subdict by key "DEFAULT_HANDLERS_KEY"
-If this handler is custom for current model you shuld add it in COLUMN_HANDLERS
-dict by key "Model.__name__"
+If this handler is custom for current model you should add it in
+COLUMN_HANDLERS dict by key "Model.__name__"
```

Comment:
Please fix the name of the constant in docstring.
```suggestion
_COLUMN_HANDLERS dict by key "Model.__name__"
```